config/external_db_map/default.txt

```diff
@@ -88,6 +91,9 @@ GenBank	GenBank_transcript	transcript
 GB	GenBank_transcript	transcript
 GenBank_transcript	GenBank_transcript	transcript
 
+JGIDB	JGI_TRANSCRIPT	transcript
```

Comment:
Thanks !! This DB was the only failure for the 8 new genome loads Im currently running